### PR TITLE
Added in steps for Local development in README.md

### PR DESCRIPTION
**Pull request by MikAoJk**

### CODE REVIEW

**Comment by MikAoJk** [comment]
@olaven

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
 
 https://nrss.deno.dev/
+
+## Local devlopment
+install Deno
+https://deno.land/manual/getting_started/installation
+
+### Run the application
+``` shell
+deno run main.ts
```
Comment (olaven) [inline README.md:11]:
```suggestion
deno task start
```

This change uses the task in `deno.json`, meaning that it'll also update on file changes.

**Review by olaven — CHANGES_REQUESTED** [review]
Thank you for contributing 🎉 This looks great and I am happy to merge if my review comment is addressed. 

Olav

**Comment by MikAoJk** [comment]
> Thank you for contributing 🎉 This looks great and I am happy to merge if my review comment is addressed.
> 
> Olav

Ofc no problem😁

**Review by olaven — APPROVED** [review]

**Pull request merged**